frontend/model/captureLogs.js

```diff
@@ -189,5 +190,20 @@ sbp('sbp/selectors/register', {
   'appLogs/get' () { return getLogger()?.entries?.toArray() ?? [] },
   'appLogs/save' () { getLogger()?.save() },
   'appLogs/pauseCapture': captureLogsPause,
-  'appLogs/startCapture': captureLogsStart
+  'appLogs/startCapture': captureLogsStart,
+  'appLogs/logServer': process.env.NODE_ENV !== 'development' || !window.location.href.startsWith('https://gi')
+    ? noop
+    : function (level, stringifyMe) {
+      if (level === 'debug') return // comment out to send much more log info
+      const string = JSON.stringify(stringifyMe)
```

Comment:
I would like to change name from `string` to `value` since string is the name of type.